tecken/api/views.py

```diff
@@ -632,43 +632,6 @@ def nones_to_zero(obj):
     return http.JsonResponse(context)
 
 
-# Store a result for 10 minutes
-SYMINFO_RESULT_CACHE_TIMEOUT = 600
-
-# Indicates there's nothing in the cache
-NO_VALUE_IN_CACHE = object()
-
-
-@metrics.timer_decorator("syminfo.lookup.timing")
-def cached_lookup_by_syminfo(somefile, someid, refresh_cache=False):
```

Comment:
I moved this to the download view code. It's a little awkward there, too, but it seemed a better place than here.